### PR TITLE
[range.istream.view] Add \ref for `basic_istream_view::iterator`

### PR DESCRIPTION
**Pull request by hewillk**

### CODE REVIEW

File: source/ranges.tex [hunk 1]
```diff
@@ -3257,6 +3257,7 @@
     constexpr default_sentinel_t end() const noexcept;
 
   private:
+    // \ref{range.istream.iterator}, class \tcode{basic_istream_view::\exposid{iterator}}
```
Comment (CaseyCarter) [inline source/ranges.tex:3260]: Conventionally these cross-references-in-synopsis-comments repeat the title of the referenced subclause:
```suggestion
    // \ref{range.istream.iterator}, class template \tcode{basic_istream_view::\exposid{iterator}}
```

Comment (JohelEGP) [reply source/ranges.tex:3260]: Then there's an error _there_, because this `iterator` isn't preceded by a _template-head_.

**Review by CaseyCarter — APPROVED** [review]

**Comment by tkoeppe** [comment]
Thanks!

File: source/ranges.tex (hunk 1 above)
Comment (CaseyCarter) [reply source/ranges.tex:3260]: For posterity: I've submitted #5515 to fix this.

**Pull request merged**